src/test/java/org/javarosa/xpath/test/XPathEvalTest.java

```diff
@@ -58,8 +58,8 @@ private void testEval(String expr, FormInstance model, EvaluationContext ec, Obj
             if (exceptionExpected) {
                 fail("Expected exception, expression : " + expr);
             } else if ((result instanceof Double && expected instanceof Double)) {
-                Double o = (Double)result;
-                Double t = (Double)expected;
+                Double o = (Double) result;
```

Comment:
So I cannot currently find the style guide of ours that I am pretty sure exists, but our practice with casting is actually to leave no space between the closing paren and the target of the cast. Sometimes Android Studio tries to change this for you automatically which is incredibly annoying, so I'm not even sure if you changed these explicitly, but would you mind switching back the lines in this file that were changed? (Also note that I am a big fan of all other linting changes you made in this PR 😄)